carbon/carbon.py

```diff
@@ -9,20 +9,15 @@
 load_dotenv()
 
 CARBON_URL = "https://carbon.now.sh?l={language}&code={code}&bg={background}&t={theme}"
-CHROMEDRIVER_PATH = os.environ["CHROMEDRIVER_PATH"]
+CHROMEDRIVER_PATH = os.environ.get("CHROMEDRIVER_PATH", "")
 
 # in case of a slow connection it might take a bit longer to download the image
 SECONDS_SLEEP_BEFORE_DOWNLOAD = int(os.environ.get("SECONDS_SLEEP_BEFORE_DOWNLOAD", 3))
 
-DEFAULT_LANGUAGE = "python"
-DEFAULT_BACKGROUND = "#ABB8C3"
-DEFAULT_THEME = "seti"
-
-
 def _create_carbon_url(code, **kwargs: str) -> str:
-    language = kwargs.get("language") or DEFAULT_LANGUAGE
-    background = kwargs.get("background") or DEFAULT_BACKGROUND
-    theme = kwargs.get("theme") or DEFAULT_THEME
+    language = kwargs.get("language")
```

Comment:
this change seems to break the tests, please check ...

Reply:
I'll fix it; thanks for mentioning it.

It seems like a good opportunity to work with GitHub actions to run the test cases.

Reply:
Hey Bod,

I added a fixture to pass the required keyword to solve the problem, sorry I couldn't download the chromedriver and test it but I think this must solve the problem

Reply:
Thanks @GreatBahram I still get 2 failures, but I will fix those quickly on main. Thanks for your contribution!

Reply:
Thank you very much, I really like this application

Take care,

Reply:
Thanks!